modules/markup/markdown/markdown_test.go

```diff
@@ -635,7 +635,7 @@ mail@domain.com
 <a href="https://example.com/file.bin" rel="nofollow">https://example.com/file.bin</a><br/>
 <a href="/file.bin" rel="nofollow">local link</a><br/>
 <a href="https://example.com" rel="nofollow">remote link</a><br/>
-<a href="/src/file.bin" rel="nofollow">local link</a><br/>
+<a href="/file.bin" rel="nofollow">local link</a><br/>
```

Comment:
For reviewers: the old test is not right. It should be the same as line above (L636). And there are many similar cases, they are all fixed.